.github/workflows/sign-windows.yaml

```diff
@@ -3,7 +3,10 @@ on:
   workflow_call:
 jobs:
   create-windows-update:
-    name: Create Windows Update 🥩
+    name: Sign Windows Build 🥩
```

Comment:
With this change, the workflow run label will change from:
Windows Signing ✍️ / Create Windows Update 🥩

to:
Windows Signing ✍️ / Sign Windows Build 🥩

Is that the intended end result?

Reply:
Yes because the role of this job was changed so it no longer creates the patches. That was moved into a release workflow a while ago.